frontends/web/src/routes/account/summary/summarybalance.tsx

```diff
@@ -158,3 +158,55 @@ export const SummaryBalance = ({
     </div>
   );
 };
+
+
+export function LightningBalance() {
```

Comment:
There is some code duplication here, I think that it would be nice to move the logic to subscribe and fetch the LN balance inside accountsummary.tsx and pass it to the LightningBalance and SummaryBalance components. wdyt?